src/components/controls/MediaStreamControl.tsx

```diff
@@ -24,14 +49,30 @@ export class MediaStreamControl extends React.PureComponent<IMediaControlProps,
         )
     }
 
+    private getAudioEnabled(): boolean {
+        return this.props.stream.getAudioTracks()[0].enabled;
```

Comment:
add a check to make sure `getAudioTracks[0] !== undefined`

Reply:
Done